pkg/token/token.go

```diff
@@ -102,6 +102,9 @@ func GetTokenByAddress(address string) *Token {
 		addressMapLock.Unlock()
 		return &Token{Address: address, Meta: tokenMeta}
 	}
+	if err != nil{
+		log.Errorf("failed to get token metadata from alchemy, address: [%s], error: [%s]", address, err)
+	}
```

Comment:
lgtm, but just need to log error:
`log.Errorf("failed to get token metadata from alchemy, address: [%s], error: [%s]", address, err)`